### ISSUE TITLE
大佬，成功运行后看不到视频

### ISSUE THREAD

**Issue by TigerHH6866** [lipku/metahuman-stream#94]
运行后可以打开页面，输入文字后tts成功，随后就Connection state is failed和Connection state is closed了
flask_sockets.py已经改成websocket=True

(/root/autodl-tmp/nerfstream) root@autodl-container-22da11ad52-3d284594:~/autodl-tmp/metahuman-stream# python app.py --model musetalk --transport webrtc --batch_size 4 --avatar_id avator_1  --listenport 6006
add ffmpeg to path
Loads checkpoint by local backend from path: ./models/dwpose/dw-ll_ucoco_384.pth
cuda start
Namespace(pose='data/data_kf.json', au='data/au.csv', torso_imgs='', O=False, data_range=[0, -1], workspace='data/video', seed=0, ckpt='data/pretrained/ngp_kf.pth', num_rays=65536, cuda_ray=False, max_steps=16, num_steps=16, upsample_steps=0, update_extra_interval=16, max_ray_batch=4096, warmup_step=10000, amb_aud_loss=1, amb_eye_loss=1, unc_loss=1, lambda_amb=0.0001, fp16=False, bg_img='white', fbg=False, exp_eye=False, fix_eye=-1, smooth_eye=False, torso_shrink=0.8, color_space='srgb', preload=0, bound=1, scale=4, offset=[0, 0, 0], dt_gamma=0.00390625, min_near=0.05, density_thresh=10, density_thresh_torso=0.01, patch_size=1, init_lips=False, finetune_lips=False, smooth_lips=False, torso=False, head_ckpt='', gui=False, W=450, H=450, radius=3.35, fovy=21.24, max_spp=1, att=2, aud='', emb=False, ind_dim=4, ind_num=10000, ind_dim_torso=8, amb_dim=2, part=False, part2=False, train_camera=False, smooth_path=False, smooth_path_window=7, asr=False, asr_wav='', asr_play=False, asr_model='cpierse/wav2vec2-large-xlsr-53-esperanto', asr_save_feats=False, fps=50, l=10, m=8, r=10, fullbody=False, fullbody_img='data/fullbody/img', fullbody_width=580, fullbody_height=1080, fullbody_offset_x=0, fullbody_offset_y=0, avatar_id='avator_1', bbox_shift=5, batch_size=4, customvideo=False, customvideo_img='data/customvideo/img', customvideo_imgnum=1, tts='edgetts', REF_FILE=None, TTS_SERVER='http://localhost:9000', CHARACTER='test', EMOTION='default', model='musetalk', transport='webrtc', push_url='http://localhost:1985/rtc/v1/whip/?app=live&stream=livestream', listenport=6006)
reading images...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1100/1100 [00:11<00:00, 97.71it/s]
reading images...
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1100/1100 [00:00<00:00, 4487.69it/s]
start websocket server
Connection state is connecting
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 720
[INFO] push_audio 576
[INFO] push_audio 0
[INFO]tts audio stream 24000: (33984,)
[WARN] audio sample rate is 24000, resampling into 16000.
-------edge tts time:1.7777s
Connection state is failed
Connection state is closed

![image](https://github.com/lipku/metahuman-stream/assets/132218926/0d8735b5-3cd2-4d86-ae37-7f090c2dc061)

**Comment by lipku** [comment]
网络不支持p2p，用另一种模式吧

**Comment by talhadar90** [comment]
> 网络不支持p2p，用另一种模式吧

can you pls explain? i'm facing the same problem
